cmd/operator-sdk/bundle/build.go

```diff
@@ -0,0 +1,81 @@
+package bundle
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
```

Comment:
This command would be easier to manage by having these variables be struct fields:
```go
type bundleCmd struct {
	dirBuildArgs       string
	tagBuildArgs       string
	imageBuilderArgs   string
	packageNameArgs    string
	channelsArgs       string
	channelDefaultArgs string
	overwriteArgs      bool
}

func newBundleBuildCmd() *cobra.Command {
	c := bundleCmd{}
	bundleBuildCmd := &cobra.Command{
		...
	}

	bundleBuildCmd.Flags().StringVarP(&c.dirBuildArgs, "directory", "d", "", "The directory where bundle manifests are located")
	...
}
```

Since `generate` args are a subset of `build`'s, this struct can be shared between commands.